lib/sinatra/show_exceptions.rb

```diff
@@ -24,23 +24,22 @@ def call(env)
 
       if prefers_plain_text?(env)
         content_type = "text/plain"
-        exception_string = dump_exception(e)
+        exception = dump_exception(e)
       else
         content_type = "text/html"
-        exception_string = pretty(env, e)
+        exception = pretty(env, e)
       end
 
       env["rack.errors"] = errors
-
-      # Post 893a2c50 in rack/rack, the #pretty method above, implemented in
-      # Rack::ShowExceptions, returns a String instead of an array.
-      body = Array(exception_string)
+      body_array = exception.is_a?(String) ? [exception] : exception
```

Comment:
I don't think that introducing a ternary operator for something that was a simple declaration before is simpler.